tests/test_office_structure/test_get_node.py

```diff
@@ -0,0 +1,28 @@
+import json
+from tests.base import BaseTestCase
+from fixtures.office_structure.get_node_fixtures import (
+    get_node_query,
+    get_node_query_response,
+    get_node_non_existant_id_query,
+    get_node_with_non_existant_id_response
+)
+
+
+class TestQueryNode(BaseTestCase):
+    def test_get_node_query(self):
+        """
+        Test get specific node by id
+        """
+        response = self.client.execute(get_node_query)
+        actual_response = json.loads(json.dumps(response))
+        self.assertEquals(actual_response, get_node_query_response)
+
+    def test_get_node_with_non_existant_id(self):
+        """
+        Test get node with non-existant id
```

Comment:
Use a description that describes what we are testing for.

Reply:
Thanks for the feedback. This has been handled.